src/one_dragon/base/notify/push.py

```
@@ -1087,8 +1184,16 @@ def log_error(self, message: str) -> None:
 
 
     def get_config(self, key: str):
-        """获取推送配置值"""
-        return getattr(self.ctx.push_config, key.lower(), None)
+        """获取推送配置值，自动判空和类型转换"""
```

Ruff (0.12.2)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)